tests/test_0692_fsspec.py

```diff
@@ -20,7 +22,7 @@ def test_open_fsspec_http():
 def test_open_fsspec_github():
     pytest.skip("not working yet")
     with uproot.open(
-        "github://CoffeaTeam:coffea@master/tests/samples/nano_dy.root",
+        "github://scikit-hep:scikit-hep-testdata@main/src/skhep_testdata/data/uproot-issue121.root",
```

Comment:
Interesting: I didn't know about `github://`. But perhaps you should link to a tag, `v0.4.33`, instead of `main`, which could (in principle) remove this file in the future.